spinn_front_end_common/utilities/report_functions/board_chip_report.py

```diff
@@ -49,8 +49,10 @@ def _write_report(writer, machine, progress_bar):
     :param ~spinn_machine.Machine machine:
     :param ~spinn_utilities.progress_bar.ProgressBar progress_bar:
     """
-    for chip in progress_bar.over(machine.ethernet_connected_chips):
-        xys = machine.get_existing_xys_on_board(chip)
+    for e_chip in progress_bar.over(machine.ethernet_connected_chips):
+        xyps = [f"({chip.x}, {chip.y}, P: {chip.get_physical_core_id(0)})"
```

Comment:
Why the hardcode P 0?

Reply:
It is virtual processor 0 that I want to record the ID of, as that is the monitor.  If any other core crashes, we can likely ask the machine for the details, but if the monitor crashes, this will be the only record.